### PR TITLE
feat: adding motion variables

### PR DESCRIPTION
**Pull request by ckrook**
## **Describe pull-request**  
Adding the motion variables for Tegel

### CODE REVIEW

**Review by Lunkan89 — APPROVED** [review]
LGTM :)

File: motion/_motion.scss
Comment (mJarsater) [inline motion/_motion.scss:1]: Should these be demoed in storybook in any way?

Comment (timrombergjakobsson) [reply motion/_motion.scss:1]: Yes box! We will aim on merging the global variables not being tied to any usage in this PR.

Comment (mJarsater) [reply motion/_motion.scss:1]: Sure, but if these are meant to be distributed in the design system there is a value in demoing then so users could add them to their own components?

Comment (timrombergjakobsson) [reply motion/_motion.scss:1]: Ah, well, yes maybe something like that is a good idea.

File: motion/index.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@import '_motion';
```
Comment (timrombergjakobsson) [inline motion/index.scss:1]: I think we can drop the underscore.

```suggestion
@import 'motion';
@import 'animation';
```

**Review by theJohnnyMe — APPROVED** [review]
LGTM

**Review by timrombergjakobsson — APPROVED** [review]
Looks great! Merge?

**Pull request merged**